Tests/Common/Util/ExtensionsTests.cs

```diff
@@ -216,6 +217,14 @@ public void SafeDecimalCastRespectsLowerBound()
             Assert.AreEqual(decimal.MinValue, output);
         }
 
+        [Test]
+        public void NormalizeDecimalReturnsNoTrailingZeros()
+        {
+            var input = 1.72340000m;
+            var output = input.Normalize();
+            Assert.IsFalse(output.ToString(CultureInfo.InvariantCulture).EndsWith("0"));
```

Comment:
Cool, thanks mike.  I've never really used the TestCase attribute in Nunit.  Very clean!